### PR TITLE
OrbusInfinity (Certified Connector)

### PR DESCRIPTION
**Pull request by Bartek-StO**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/OrbusInfinity/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,5351 @@
...
+                    "description": "Receiver Url.",
+                    "minLength": 1,
+                    "type": "string",
+                    "x-ms-notification-url": true,
+                    "x-ms-visibility": "internal"
+                },
+                "expirationDate": {
+                    "format": "date-time",
+                    "description": "Gets or sets the expiration date. If no expiration date is provided, one value\r\nby default will be used (tipically 6 months after creation date).",
+                    "type": "string"
```
Comment (vmanoharas) [inline certified-connectors/OrbusInfinity/apiDefinition.swagger.json:1634]: Please correct the spelling tipically

File: certified-connectors/OrbusInfinity/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
+## Orbus Infinity Connector
+
```
Comment (vmanoharas) [inline certified-connectors/OrbusInfinity/README.md:3]: Please update Readme.md file as per the template mentioned below. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

File: certified-connectors/OrbusInfinity/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oAuthSetting",
```
Comment (vmanoharas) [inline certified-connectors/OrbusInfinity/apiProperties.json]: Please modify from oAuthSetting to oauthSetting.  Kindly refer the schema here. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/paconn-apiProperties.schema.json

File: certified-connectors/OrbusInfinity/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+            "EnableOnbehalfOfLogin": {
+              "value": false
+            }
+          },
+          "clientId": "<<Please add your clientID here>>"
+        },
+        "uiDefinition": null
+      },
+      "token:tenantId": {
+        "type": "string",
```
Comment (vmanoharas) [inline certified-connectors/OrbusInfinity/apiProperties.json:32]: Please fix the below error, kindly refer the schema here for help. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/paconn-apiProperties.schema.json

##[error]@{level=Error; message=Property 'metadata' has not been defined and the schema does not allow additional properties.; valuePath=line: 34 position: 19 path: '[0].properties.connectionParameters.token:tenantId.metadata'; value=metadata}

File: certified-connectors/OrbusInfinity/apiProperties.json [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+        "metadata": {
+          "sourceType": "AzureActiveDirectoryTenant"
+        },
+        "uiDefinition": {
+          "constraints": {
+            "required": "false",
+            "hidden": "true"
```
Comment (vmanoharas) [inline certified-connectors/OrbusInfinity/apiProperties.json:39]: this should be "hidden": true

File: certified-connectors/OrbusInfinity/apiProperties.json [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+            "TenantId": {
+              "value": "common"
+            },
+            "EnableOnbehalfOfLogin": {
+              "value": false
+            }
+          },
+          "clientId": "<<Please add your clientID here>>"
+        },
+        "uiDefinition": null
```
Comment (vmanoharas) [inline certified-connectors/OrbusInfinity/apiProperties.json]: Please fix the below error, kindly refer this schema and update https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/paconn-apiProperties.schema.json

@{level=Critical; message=Invalid type for the UIDefinition object.; valuePath=$.properties.connectionParameters.token.uiDefintion; value=}

**Review by vmanoharas — COMMENTED** [review]
Hello @Bartek-StO, 

Kindly resolve review comments.

**Comment by Bartek-StO** [comment]
@microsoft-github-policy-service agree company="Orbus Software"

File: certified-connectors/OrbusInfinity/apiProperties.json (hunk 3 above)
Comment (Bartek-StO) [reply certified-connectors/OrbusInfinity/apiProperties.json]: fixed

File: certified-connectors/OrbusInfinity/apiDefinition.swagger.json (hunk 1 above)
Comment (Bartek-StO) [reply certified-connectors/OrbusInfinity/apiDefinition.swagger.json:1634]: fixed

File: certified-connectors/OrbusInfinity/apiProperties.json (hunk 4 above)
Comment (Bartek-StO) [reply certified-connectors/OrbusInfinity/apiProperties.json:32]: fixed

File: certified-connectors/OrbusInfinity/apiProperties.json (hunk 6 above)
Comment (Bartek-StO) [reply certified-connectors/OrbusInfinity/apiProperties.json]: fixed

File: certified-connectors/OrbusInfinity/apiProperties.json (hunk 5 above)
Comment (Bartek-StO) [reply certified-connectors/OrbusInfinity/apiProperties.json:39]: When I try to put true without double quote got problem 
2024-01-11_14h58_43

File: certified-connectors/OrbusInfinity/README.md (hunk 2 above)
Comment (Bartek-StO) [reply certified-connectors/OrbusInfinity/README.md:3]: updated

**Review by vmanoharas — APPROVED** [review]
Hello @Bartek-StO 
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**